common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -99,3 +99,18 @@
 else:
     tmpl.render_context(context)
 %></%def>
+
+<%def name="optional_include_mako_with_fallback(file, fallback, with_microsite=False, **kwargs)"><%
```

Comment:
You're right that I suggested that in chat, but I was throwing out a suggestion without really considering the implications. Now that I see the code, I think it would be better to fold this improvement into the original function. (Sometimes my off-the-cuff suggestions are wrong. Sorry! :stuck_out_tongue_winking_eye:)

You're right that one fallback template will probably be sufficient in many cases, but probably not *all* cases. We should consider how we want to handle those other cases. I'm open to suggestions -- maybe we should optimize for the "one fallback" case, and have a separate argument that is used for multiple fallbacks. Maybe we shouldn't handle multiple fallbacks at all, and let the template author write the logic into the template. But I suspect there's something we can do to make it at least a *little* bit cleaner.